appserver/tests/payara-samples/samples/client-certificate-validator/src/test/resources/META-INF/services/fish.payara.security.api.ClientCertificateValidator

```diff
@@ -0,0 +1 @@
+fish.payara.security.client.CertificateValidator
```

Comment:
Isn't this the wrong way around. The file needs to be named like the interface and the file contains the name of the implementation class?

Reply:
yes, it is named exactly that. I think you are confused. tests wouldn't pass otherwise :)

Reply:
Sorry. You are right. I was confused by the names. I instictively thought the more general name is supposed to be the interface and the more specific name is the implementation.